### ISSUE TITLE
`jsx-a11y/anchor-has-content`: do not encourage title attribute usage

### ISSUE THREAD

**Issue by mikemai2awesome** [jsx-eslint/eslint-plugin-jsx-a11y#1060]
From: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-has-content.md

> Alternatively, you may use the `title` prop or the `aria-label` prop.

Can we please remove the mention of the HTML `title` attribute? It is problematic with assistive tech. `aria-label` and `aria-labelledby` should be the only alternatives accepted. [[source](https://www.tpgi.com/using-the-html-title-attribute/)]

Then remove this from the success criteria.

```html
<a title='foo' />
```

**Comment by ljharb** [comment]
Do you have a more recent source than 15 years ago for this?

**Comment by TMaszko** [comment]
Could you assign me to this one ?

**Comment by mikemai2awesome** [comment]
@smellai i think that makes a lot of sense.
